packages/opentelemetry-resources/src/platform/node/service-name.ts

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
this might be tricky the path might be a different depends how you run the final app. Also people might be using bundlers and not always the package.json will be available.
Why not simply reusing the script for generating `version.ts` file and just add additional things there like `name`; Then it will always be bundled together with the rest.

Reply:
The resource isn't the instrumentation, but the service being instrumented. If the package.json is not found, it falls back to using the command line which started the program.